python/ray/workflow/tests/test_http_events.py

```diff
@@ -0,0 +1,142 @@
+import asyncio
```

Comment:
we also need to test the negative case and also make sure checkpoint is done before return.

Reply:
Negative cases added to test_http_events.py